File: scratch/change_colors.py
import os

files = [
    'src/routes/_authenticated/iluminacao.tsx',
    'src/routes/_authenticated/iluminacao.$evento_id.tsx'
]

for filepath in files:
    with open(filepath, 'r', encoding='utf-8') as f:
        content = f.read()
    
    content = content.replace('yellow-', 'amber-')
    
    with open(filepath, 'w', encoding='utf-8') as f:
        f.write(content)

print("Colors updated.")

File: scratch/change_colors.js
const fs = require('fs');

const files = [
  'src/routes/_authenticated/iluminacao.tsx',
  'src/routes/_authenticated/iluminacao.$evento_id.tsx'
];

files.forEach(file => {
  let content = fs.readFileSync(file, 'utf8');
  content = content.replace(/yellow-/g, 'amber-');
  fs.writeFileSync(file, content);
});

console.log("Colors updated.");
